### PR TITLE
Add popup modal to the desktop and mobile version of the portfolio

### PR DESCRIPTION
**Pull request by tariktesfa**
In this pull request, we have added the popup to the portfolio project detail by using javascript objects. The following files are added to the repo

- main.js to add the popup
- index.html to add the elements of the portfolio'
- desktop.css to style the desktop version of the portfolio
- style.css to style the mobile version of the portfolio

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -7,4 +7,177 @@ const closeMobileMenu = () => {
...
+  imageMobile: './images/Snapshoot3.png',
+  imagePopup: './images/Snap3.png',
+  imageMobilePopup: './images/Snapshoot3.png',
+  seeLive: 'https://tariktesfa.github.io/My-portfolio/',
+  seeSource: 'https://github.com/tariktesfa/My-portfolio',
+},
+{
+  title: 'Uber Navigation',
+  exprience: ['UBER', 'Lead Developer', '2018'],
+  desription: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
```
Comment (rotimiazeez) [inline js/main.js:57]: - [ ] Please make sure the popup follows the design for both desktop and mobile version from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins)😉.
- Your popup for mobile
screenshot (14)
- Figma popup for mobile
screenshot (15)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM👋, 

## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 1 above)
Comment (tariktesfa) [reply js/main.js:57]: I have fixed the issue with the mobile view.

**Review by oxenprogrammer — APPROVED** [review]
Hi @tariktesfa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**